The check error (linting) is not related to this PR:

> ./scripts/check_pydantic.sh .
> ./scripts/lint_imports.sh
> poetry run ruff .
> [ "langchain_community" = "" ] || poetry run ruff format langchain_community --diff
> 966 files already formatted
> [ "langchain_community" = "" ] || poetry run ruff --select I langchain_community
> [ "langchain_community" = "" ] || mkdir -p .mypy_cache && poetry run mypy langchain_community --cache-dir .mypy_cache
> langchain_community/utilities/requests.py:144: note: Maybe you forgot to use "await"?
> langchain_community/utilities/requests.py:144: note: Error code "misc" not covered by "type: ignore" comment
> langchain_community/vectorstores/thirdai_neuraldb.py:29: error: Function is missing a type annotation  [no-untyped-def]
> Found 1 error in 1 file (checked 966 source files)
> make: *** [Makefile:50: lint_package] Error 1
> Error: Process completed with exit code 2.
>

--- a/libs/core/langchain_core/runnables/__init__.py
+++ b/libs/core/langchain_core/runnables/__init__.py
@@ -9,7 +9,9 @@ synchronous, asynchronous, batch, and streaming operations.
 Support for **async** allows servers hosting LCEL based programs to scale better
 for higher concurrent loads.
 
-**Streaming** of intermediate outputs as they're being generated allows for
+**Batch** operations allow for processing multiple inputs in parallel.
+
+**Streaming** of intermediate outputs, as they're being generated, allows for
 creating more responsive UX.
 
 This module contains schema and implementation of LangChain Runnables primitives.

--- a/libs/langchain/langchain/runnables/__init__.py
+++ b/libs/langchain/langchain/runnables/__init__.py
@@ -1,0 +1,18 @@
+"""LangChain **Runnable** and the **LangChain Expression Language (LCEL)**.
+
+The LangChain Expression Language (LCEL) offers a declarative method to build
+production-grade programs that harness the power of LLMs.
+
+Programs created using LCEL and LangChain Runnables inherently support
+synchronous, asynchronous, batch, and streaming operations.
+
+Support for **async** allows servers hosting LCEL based programs to scale better
+for higher concurrent loads.
+
+**Batch** operations allow for processing multiple inputs in parallel.
+
+**Streaming** of intermediate outputs, as they're being generated, allows for
+creating more responsive UX.
+
+This module contains non-core Runnable classes.
+"""

--- a/libs/langchain/langchain/runnables/__init__.py
+++ b/libs/langchain/langchain/runnables/__init__.py
@@ -6,8 +6,8 @@ production-grade programs that harness the power of LLMs.
 Programs created using LCEL and LangChain Runnables inherently support
 synchronous, asynchronous, batch, and streaming operations.
 
-Support for **async** allows servers hosting LCEL based programs to scale better
-for higher concurrent loads.
+Support for **async** allows servers hosting the LCEL based programs
+to scale better for higher concurrent loads.
 
 **Batch** operations allow for processing multiple inputs in parallel.
 

